src/components/Collections/Collections.js

```diff
@@ -25,13 +43,43 @@ class Collections extends React.Component {
 
   render() {
     const { collections } = this.state;
+
     return (
       <>
-        {collections && collections.length === 0 && (
-          <div>This user does not have any collections.</div>
-        )}
+        <LoadingBar color="#6958f2" ref={this.loadingBar} />
+        {collections && collections.length === 0 && <div>This user does not have any collections.</div>}
         {collections && collections.length > 0 && (
-          <div>Collections go here</div>
+          <Container>
+            {collections.map((collection) => {
+              let imageSrc = "";
+              if (!collection.cover_photo) {
+                imageSrc = unavilableCover;
+              } else if (collection.cover_photo) {
+                imageSrc = collection.cover_photo.urls.small;
+              }
```

Comment:
`const imageSrc = collection.cover_photo ? collection.cover_photo.urls.small : unavilableCover`

Reply:
It always seems so obvious after you point it out.